python/pypto/runtime/execute_artifact.py

```
@@ -0,0 +1,158 @@
+        enable_dep_gen=args.enable_dep_gen,
+        enable_scope_stats=args.enable_scope_stats,
+    )
+    try:
+        execute_artifact_dir(
+            args.work_dir,
+            args.platform,
+            args.device_id,
+            pto_isa_commit=args.pto_isa_commit,
+            dfx=dfx,
+        )
+    except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)